### ISSUE TITLE
WM_MT_button_context deprecated in Blender 3.3

### ISSUE THREAD

**Issue by cl3m3c7** [Mustard2/MustardUI#97]
when closing blender this exception raises:
```python
Exception in module unregister(): '~/.config/blender/3.4/scripts/addons/mustard_ui.py'
Traceback (most recent call last):
  File "/blender/3.4/scripts/modules/addon_utils.py", line 421, in disable
    mod.unregister()
  File "~/.config/blender/3.4/scripts/addons/mustard_ui.py", line 9733, in unregister
    bpy.types.WM_MT_button_context.remove(mustardui_property_menuadd)
AttributeError: 'module' object has no attribute 'WM_MT_button_context'
```

> The API for adding context menu entries has changed. Instead of manually registering `WM_MT_button_context`, draw functions should be appended to `UI_MT_button_context_menu`, which is registered by Blender itself. The old API is still available for compatibility, but is deprecated and will be removed in a future release, therefore addons should update to the new API.

for reference: [https://wiki.blender.org/wiki/Reference/Release_Notes/3.3/Python_API](https://wiki.blender.org/wiki/Reference/Release_Notes/3.3/Python_API)

**Comment by Mustard2** [comment]
Nice to know, I completely missed this, thank you!
I'll fix it in the next version

**Comment by Mustard2** [comment]
Fixed in #96
